ClassTranscribeServer/Controllers/MediaController.cs

```diff
@@ -58,6 +58,16 @@ public async Task<ActionResult<MediaDTO>> GetMedia(string id)
                 }
             }
 
+            var visibilityAuthorizationResult = await _authorizationService.AuthorizeAsync(User, media.Playlist.Offering, Globals.POLICY_UPDATE_OFFERING);
+            if (!visibilityAuthorizationResult.Succeeded)
+            {
+                if (media.Visibility == Visibility.Hidden || media.Playlist.Visibility == Visibility.Hidden || media.Playlist.Offering.Visibility == Visibility.Hidden)
```

Comment:
This line could throw an error, if it does. The problem should be that the related objects have to be loaded. (i.e. media.Playlist and media.Playlist.Visibility) 

You'll have to do something like this https://github.com/classtranscribe/WebAPI/blob/a117f169872cbef1ef6ff142d38b5160a2295804/TaskEngine/Tasks/ProcessVideoTask.cs#L30 

Only if it fails.

Reply:
I've tested those api and fixed some bugs. Here the loading of [IgnoreDataMember] works well without 'Include' so I didn't make any changes.